packages/mui-material/src/styles/useThemeProps.js

```diff
@@ -1,6 +1,6 @@
 import { useThemeProps as systemUseThemeProps } from '@mui/system';
 import defaultTheme from './defaultTheme';
 
-export default function useThemeProps({ props, name }) {
-  return systemUseThemeProps({ props, name, defaultTheme });
+export default function useThemeProps({ props, name, mergePropsWhiteList }) {
```

Comment:
We would also need to update the types in declaration files (.d.ts) in MUI System's `useThemeProps` and Material UI's `useThemeProps` methods. Also in `getThemeProps` in MUI System.

Reply:
The types were added.